### PR TITLE
doc(ios/info): add needed NSPhotoLibraryUsageDescription information

### PR DESCRIPTION
**Pull request by matheusdavidson**
Update instructions for IOS to show the needed NSPhotoLibraryUsageDescription key on info.plist.
#38

### CODE REVIEW

**Comment by matheusdavidson** [comment]
This is how it looks:
image

File: README.md [hunk 1]
```diff
@@ -109,6 +109,16 @@ album: this.platform.is('ios') ? album.identifier : album.name;
 - `npx cap add ios`
 - `npx cap open ios`
 - sign your app at xcode (general tab)
+- add the following key to `Info.plist.`:
+
+```xml
+<dict>
+  ...
+  <key>NSPhotoLibraryUsageDescription</key>
+  <string>Describe why you need access to the library</string>
```
Comment (nkalupahana) [inline README.md:118]:
```suggestion
  <string>Describe why you need access to user's photos</string>
```

File: README.md [hunk 2]
```diff
@@ -109,6 +109,16 @@ album: this.platform.is('ios') ? album.identifier : album.name;
 - `npx cap add ios`
 - `npx cap open ios`
 - sign your app at xcode (general tab)
+- add the following key to `Info.plist.`:
```
Comment (nkalupahana) [inline README.md:112]:
```suggestion
- add the following to `Info.plist`:
```

**Review by nkalupahana — CHANGES_REQUESTED** [review]
A few small changes -- thank you!

**Review by nkalupahana — APPROVED** [review]

**Pull request merged**